README.MD

```diff
@@ -0,0 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Mobile Menu
+
+In this project i implemented the design of the navbar in its mobile version.
+
+## Built With
+
+- JavaScript
+- CSS
+
+## Screenshot
+
+![Screenshot](screenshot1.png)
+
```

Comment:
Great work adding a link to the screenshot 👏🏿 . However, although you have attempted to add a screenshot just like suggested by the previous reviewer, the link is broken. It will be great if you could fix this issue as it will make your project even more professional